home/src/main/resources/rdf/i18n/pt_BR/interface-i18n/firsttime/vitro_UiLabel.ttl

```diff
@@ -254,6 +254,78 @@ uil-data:reset_password_note.Vitro
         uil:hasKey      "reset_password_note" ;
         uil:hasPackage  "Vitro-languages" .
 
+uil-data:password_reset_email_sent.Vitro
+        rdf:type         owl:NamedIndividual ;
+        rdf:type         uil:UILabel ;
+        rdfs:label       "Um e-mail de recuperação de senha será enviado para a conta {0} se essa estiver vinculado a uma conta existente."@pt-BR ;
+        uil:hasApp       "Vitro" ;
+        uil:hasKey       "password_reset_email_sent" ;
+        uil:hasPackage   "Vitro-languages" .
+
+uil-data:password_reset_too_many_requests.Vitro
```

Comment:
Not used anywhere in the code, should be removed

Reply:
Are you going to remove it?

Reply:
I thought I did allready, now it is done.